frida_mode/GNUmakefile

```diff
@@ -364,6 +394,7 @@ $(AFLPP_QEMU_DRIVER_HOOK_OBJ): $(AFLPP_QEMU_DRIVER_HOOK_SRC) | $(BUILD_DIR)
 hook: $(AFLPP_FRIDA_DRIVER_HOOK_OBJ) $(AFLPP_QEMU_DRIVER_HOOK_OBJ)
 
 ############################# ADDR #############################################
+ifneq "$(OS)" "android"
```

Comment:
All these individual options should probably be split into a separate env variable definition.
Then you can just have...
```
ifneq "$(OS)" "android"
  MY_ENV+=-lrt
endif
````